### ISSUE TITLE
The Bar RP/Hentai Bots Are Hard to Serve On Delta Station

### ISSUE THREAD

**Issue by Pugie** [tgstation/tgstation#57388]
Let me explain:

If you place a drink to the south of the robot within 1 tile (i.e. the bar/kitchen counters on metastation) they will pick up their drink and dispense the money on the counter.

On Delta station, however, the bar is built in the north of the service room, so the bots can't pick up the drinks and you have to go all the way through theatre to the other side of the bar to serve them.

Basically, the hentai/brazzerbots can't accept food and drinks north of them, only south of them.

**Comment by TheBonded** [comment]
why can't they just accept drinks anywhere in the tiles around them? or even seek out drinks they can pathfind to?

**Comment by MrMelbert** [comment]
> On Delta station, however, the bar is built in the north of the service room, so the bots can't pick up the drinks and you have to go all the way through theatre to the other side of the bar to serve them.

This is because the stools are backwards, alt-click them twice and they'll rotate the right way around and you don't need to go around the counter

**Comment by Pugie** [comment]
> > On Delta station, however, the bar is built in the north of the service room, so the bots can't pick up the drinks and you have to go all the way through theatre to the other side of the bar to serve them.
> 
> This is because the stools are backwards, alt-click them twice and they'll rotate the right way around and you don't need to go around the counter

hahahahahahahahahhahahahahaha

**Comment by cacogen** [comment]
It's based on the stool's direction? Who could know? @TheBonded is right

**Comment by Qustinnus** [comment]
performance

**Comment by cacogen** [comment]
Maybe the decals that are placed by the seating thing could have an arrow indicating the stool's direction and an examine hint to rotate the stool.
